### ISSUE TITLE
Nemo Preview - Bad Configuration

### ISSUE THREAD

**Issue by Jopp-gh** [linuxmint/nemo-extensions#532]
**Nemo preview** is quick, useful and not to forget, awesome.

The current Configuration however, is messed up:

- the window isn't centered, but placed with a shift to the bottom right. (Allow me to say, **Centered** IS neutral, everything else IS "custom")
- the window size itself is insufficient. Rendering of previews is quite useless. Nemo preview can preview anything, and shouldn't be limited to view icons only. Nemo preview needs to be configured correctly so we can preview A4 office documents and photography.

### Solution
To sum up:

1. place width formats (presentations, landscape photography, etc) at the absolute center (vertical and horizontal center) so files can cover the whole width on screen. 
2. Place height formats (office documents, portrait photography, etc) at the vertical center on screen so files can cover the whole height on screen
